ironfish/src/network/peerNetwork.ts

```diff
@@ -1242,8 +1242,8 @@ export class PeerNetwork {
       return
     }
 
-    // add the block to the chain
-    const result = await this.node.syncer.addBlock(peer, block)
+    // add the block to the chain, skipping verification
+    const result = await this.node.syncer.addBlock(peer, block, true)
```

Comment:
verifyBlock is called above. When `syncer.addBlock` is called, then the following code is run: 

```
    const { isAdded, reason, score } = await this.chain.addBlock(block)
``` 

then `connect` is called in the Blockchain class when then calls: 

```
    const verifyBlockAdd = this.verifier.verifyBlock(block, { prev }).catch((_) => {
      return { valid: false, reason: VerificationResultReason.ERROR }
    })
```